src/hooks/login/useLedgerLogin.ts

```diff
@@ -61,6 +60,7 @@ export function useLedgerLogin({
   onLoginRedirect
 }: UseLedgerLoginPropsType): LedgerLoginHookReturnType {
   const ledgerAccount = useSelector(ledgerAccountSelector);
+  const hwProvider = getAccountProvider() as HWProvider;
```

Comment:
why cast?

Reply:
`getAccountProvider()` has multiple types and dynamic cast is required

Reply:
Resolved